httpx/client.py

```diff
@@ -237,6 +237,33 @@ def _get_auth_middleware(
 
         return None
 
+    def build_request(
```

Comment:
is `build_request` func's position correct in BaseClient ?

Reply:
I asked to understand if BaseClient is an abstract class for all custom client classes. so far it is used to narrow `Client` and `AsyncClient` code down.

Reply:
I think so
`Client` and `AsyncClient` build Request in same way